ansible/group_vars/all/piecewise_global_config.yml

```diff
@@ -125,7 +125,7 @@ map_bounding_box: "{{ map_coordinate_southwest.lon }},{{ map_coordinate_southwes
 # it is advisable to set 'map_data_data_end_date' to a date well in the future, for
 # example: "Dec 31 2050 00:00:00"
 map_data_start_date: "Jan 1 2017 00:00:00"
-map_data_end_date: "Feb 15 2017 00:00:00"
+map_data_end_date: "Mar 13 2017 00:00:00"
```

Comment:
This isn't all that important now, since this is still in testing, but is there a reason to have `map_data_end_date` set to a date in the past rather than "well in the future"?